larray_editor/editor.py

```diff
@@ -275,12 +278,22 @@ def setup_menu_bar(self):
         file_menu.addSeparator()
         file_menu.addAction(create_action(self, _('&Quit'), shortcut="Ctrl+Q", triggered=self.close))
 
+        ###############
+        #  Help Menu  #
+        ###############
         help_menu = menu_bar.addMenu('&Help')
         help_menu.addAction(create_action(self, _('Online &Documentation'), shortcut="Ctrl+H",
                                           triggered=self.open_documentation))
         help_menu.addAction(create_action(self, _('Online &Tutorial'), triggered=self.open_tutorial))
         help_menu.addAction(create_action(self, _('Online Objects and Functions (API) &Reference'),
                                           triggered=self.open_api_documentation))
+        help_menu.addSeparator()
+        help_menu.addAction(create_action(self, _('Report &Issue...'), triggered=self.report_issue))
+        help_menu.addAction(create_action(self, _('&Support...'), triggered=self.open_support))
```

Comment:
I do not really like "support", because **I** understand it as implying we will answer all questions there. I would rather have it named "Users forum" or "Users group" or "Users discussion" or ...

Reply:
OK for "Users discussion"